AsyncDisplayKit/ASDisplayNode.mm

```diff
@@ -1383,51 +1392,28 @@ - (void)recursivelyDisplayImmediately
   [self displayImmediately];
 }
 
-//Calling this with the lock held can lead to deadlocks. Always call *unlocked*
-- (void)__setNeedsLayout
+- (void)invalidateCalculatedLayout
```

Comment:
Currently we cannot get rid of `invalidateCalculatedLayout` as we use it in `ASEditableTextNode` the only place ... we could use it to framework internal.

I looked briefly in `ASEditableTextNode` and we do some fancy stuff in there that would break if we call `setNeedsLayout` instead of `invalidateCalculatedLayout`.

That said eventually I would like to  get rid of `invalidateCalculatedLayout`